libs/sx1302_hal/patches/001-add_compatibility_with_basicstation.patch

```diff
@@ -0,0 +1,10 @@
+--- a/libloragw/inc/loragw_hal.h
++++ b/libloragw/inc/loragw_hal.h
```

Comment:
Missing Git header, which contains usual information like: who is the author, the commit subject, commit description and from that, we can see if this is a hack or it could be upstreamed.